### DISCUSSION TITLE
Is it possible to set a radius for grid path?

### DISCUSSION THREAD

**Discussion by tejab92** [Eliav2/react-xarrows#65] [Q&A]
![image](https://user-images.githubusercontent.com/82674602/116315950-1d98fb80-a766-11eb-90a1-a346c98ddbb8.png)

**Comment by tejab92** [comment]
@Eliav2

**Reply by jithureddy** [reply]
@tejab92 Nice request. The entire UX would be enhanced when we can set the radius for grid path. @Eliav2 Really good effort to make this lib !!

**Reply by Eliav2** [reply]
will be available on the next release. the entire library has rewritten

**Reply by olup** [reply]
So is it available now ?
